tests/integration/cases/basic/test_basic_commands.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from twisted.internet import defer
+
+import manof
+import core
+import tests.integration
+
+
+class BasicCommandsTestCase(tests.integration.ManofIntegrationTestCase):
+
+    def manof_args(self):
+        return {
+            'parallel': None,
+            'repository': None,
+            'tag_local': None,
+            'dry_run': False
+        }
+
+    def test_load_manofest(self):
+        self._logger.info('Testing load manofest happy flow')
+        expected_dependent_targets = ['load_test_image']
+
+        # load manofest
+        manofest = self._load_manofest_targets('LoadTestImage')
+
+        self.assertEqual(type(manofest), core.RootTarget)
+        self.assertEqual(manofest.to_dict()['dependent_targets'], expected_dependent_targets)
+
+    @defer.inlineCallbacks
+    def test_pull_images(self):
+        self._logger.info('Testing pull images happy flow')
+
+        # remove Test Image from docker for the test
+        self._logger.debug('Removing ubuntu:19.10 for pull test')
+        yield manof.utils.execute('docker rmi -f ubuntu:19.10',
```

Comment:
Before doing it,  resolve the `image_name` from manofest and use it 

```suggestion
        yield manof.utils.execute('docker rmi -f ubuntu:19.10',
```

Reply:
Also don't hard code image name, put it in a var / field for reuse